elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -38,33 +44,97 @@
             max-width: 1140px;
         }
     }
+
     [slot="tray"] {
         width: 100%;
+        min-height: 60px;
     }
-    .pfe-navigation-item__tray--container {
-        margin: 0 auto;
+
+    // Override grid so that CTAs do not overlap.
+    .pfe-navigation--footer:not(:empty) {
+        display: block !important;
+    }
+
+    @media screen and (min-width: $pfe-global--grid-breakpoint--lg) {
+        .pfe-navigation-item__tray--container {
+            margin: 0 auto;
+        }
+    }
+
+    [is_nested][parent_hidden] .pfe-navigation-item__tray {
+        max-height: none;
+        overflow: visible;
+    }
+
+    @media screen and (max-width: 1023px) {
```

Comment:
@chrisdo1 can we bump up this breakpoint a bit, the links are crowded until like 1100px or so
https://web-dev-drupal-webux2.dev.a1.vary.redhat.com/es/selecciona-un-idioma